### DISCUSSION TITLE
Diatone Mamba F722 V3 support?

### DISCUSSION THREAD

**Discussion by visioneerone** [iNavFlight/inav#6666] [Q&A]
I see that Diatone has a new version of their F722 mini FC out.  Among other changes it explicitly adds dedicated I2C pads, so this should fit nicely into some of the new mini-sized long range quads.

Does anyone know if the existing MAMBAF722 target can be used?

**Comment by JulioCesarMatias** [comment]
yes it can be used, but nothing has been added to the new I2C pad
